src/index.js

```diff
@@ -3,51 +3,71 @@ import { createFilter } from '@rollup/pluginutils'
 export default function css(options = {}) {
   const filter = createFilter(options.include || ['**/*.css'], options.exclude)
   const styles = {}
-  const order = []
   let dest = options.output
-  let changes = 0
+  let hasChanged = false
+  let prevIds = []
 
   return {
     name: 'css',
+    buildStart() {
+      hasChanged = false
+    },
     transform(code, id) {
       if (!filter(id)) {
         return
       }
 
+      const { imports, codeWithoutImports } = splitImports(code);
+
       // When output is disabled, the stylesheet is exported as a string
       if (options.output === false) {
+        if (imports.length === 0) {
+          return {
+            code: `export default ['${JSON.stringify(code)}'`,
```

Comment:
Could this be a typo?
```suggestion
            code: `export default ${JSON.stringify(code)}`,
```